### PR TITLE
changed dark mode nav to look more like light mode nav with shadow, c…

### PR DESCRIPTION
**Pull request by Eeshau**
changed dark mode nav to look more like light mode nav with shadow, changed full screen menu to look more like light mode, changes in some line breaks for mobile responsiveness, changed gradient on video carousel to be more subtle, changed nav logo size so it's not as squished on mobile

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -239,7 +239,7 @@ export default function Home() {
           <Grid item xs={12} md={6}>
             <Grid
               container
-              sx={{ paddingBottom: "70px", overflow: "visable" }}
+              sx={{ paddingBottom: "50px", overflow: "visable" }}
```
Comment (wryonik) [inline app/page.tsx]: `overflow: "visable"` this is not going to work due to spelling mistake :P

File: app/page.tsx [hunk 2]
```diff
@@ -311,8 +311,8 @@ export default function Home() {
                 paddingBottom: "30px",
               }}
             >
-              We leverage what emails already do... <br />
-              but using Regex & Zero Knowledge
+            We leverage what emails already do... <span className="hidden sm:inline"><br /></span>
```
Comment (wryonik) [inline app/page.tsx:314]: What is happening here?

File: app/theme.tsx [hunk 3]
```diff
@@ -61,7 +61,7 @@ const lightPalette = {
   primary: {
     main: "#FFBB6B",
     light: "#FFD580",
-    dark: "#B3773E",
+    dark: "#000000",
```
Comment (wryonik) [inline app/theme.tsx:64]: These changes seems wrong. Also where are you using them as the only place they are going to have any effect are the buttons and those also have custom colors? 

I just checked, and they are affecting the sorting buttons on the blog page in light mode. Please fix that and check other areas as well.

File: components/Accordion/Accordion.tsx [hunk 4]
```diff
@@ -1,57 +1,64 @@
...
     theme.palette.mode === "light"
+
       ? "1px solid rgba(0, 0, 0, 0.12)"
+
       : "1px solid rgba(255,255,255, 0.12)",
+
   "& .MuiAccordionSummary-root": {
+
     backgroundColor: "rgba(0, 0, 0, 0)",
+
```
Comment (wryonik) [inline components/Accordion/Accordion.tsx]: Why were these lines added? Can you run the linter again? I noticed many unintentional linting changes in other files too

File: components/Accordion/Accordion.tsx [hunk 5]
```diff
@@ -85,17 +90,19 @@ const Accordion: FC<AccordionProps> = ({
...
-            color: theme.palette.text.secondary,
-            textAlign: alignment === "right" ? "right" : "left",
-          }}
-        >
+            variant="body2"
+            sx={{
+              paddingLeft: '0px',
+              marginLeft: '0px',
+              color: theme.palette.text.secondary,
+              textAlign: alignment === "right" ? "right" : "left",
```
Comment (wryonik) [inline components/Accordion/Accordion.tsx:90]: `textAlign: alignment === "right" ? "right" : "left",` -> `textAlign: alignment`

File: components/Accordion/Accordion.tsx [hunk 6]
```diff
@@ -85,17 +90,19 @@ const Accordion: FC<AccordionProps> = ({
...
+          >
           {contents}
         </Typography>
       </AccordionDetails>
     </CustomAccordion>
   );
 };
 
-export default Accordion;
+export default Accordion;
```
Comment (wryonik) [inline components/Accordion/Accordion.tsx]: add empty line

File: components/CustomButton/CustomButton.tsx [hunk 7]
```diff
@@ -119,7 +119,7 @@ const CustomButton: React.FC<CustomButtonProps> = ({
       {...props}
       buttonLabel={buttonLabel}
     >
-      <Typography>{buttonLabel}</Typography>
+      {buttonLabel}
```
Comment (wryonik) [inline components/CustomButton/CustomButton.tsx:122]: Why did you remove Typography?

File: components/Hero/Hero.tsx [hunk 8]
```diff
@@ -179,7 +171,7 @@ export default function Hero() {
           }}
           className="pb-3 pt-3 sm:pt-4 md:pt-6 lg:pt-7 text-[12px] md:text-[20px] leading-[160%] sm:leading-[140%]"
         >
-          Anonymously prove any subset of any email <br></br>you sent or
+          Anonymously prove any subset of any email <span className="hidden sm:inline"><br /></span> you sent or
```
Comment (wryonik) [inline components/Hero/Hero.tsx:155]: same as above

File: components/Navbar/Navbar.tsx [hunk 9]
```diff
@@ -87,7 +87,7 @@ const Navbar: React.FC<NavbarProps> = ({
               justifyContent: "space-between",
             }}
           >
-            <Box sx={{ display: { xs: "flex" } }}>
+            <Box sx={{ minWidth:"70px" }}>
```
Comment (wryonik) [inline components/Navbar/Navbar.tsx]: px -> rem for widths

**Review by wryonik — CHANGES_REQUESTED** [review]

File: app/page.tsx (hunk 2 above)
Comment (Eeshau) [reply app/page.tsx:314]: removing the line break on extra small screens but keeping it on screens small and above - for mobile responsiveness

File: components/Accordion/Accordion.tsx (hunk 5 above)
Comment (Eeshau) [reply components/Accordion/Accordion.tsx:90]: makes sense fixing

File: components/Hero/Hero.tsx (hunk 8 above)
Comment (Eeshau) [reply components/Hero/Hero.tsx:155]: removing the line break on extra small screens but keeping it on screens small and above - for mobile responsiveness

File: app/page.tsx (hunk 1 above)
Comment (Eeshau) [reply app/page.tsx]: fixing!

File: components/CustomButton/CustomButton.tsx (hunk 7 above)
Comment (Eeshau) [reply components/CustomButton/CustomButton.tsx:122]: I define the button fontsize in the custom button component but wrapping the button label with a typography tag again applies the typography fontsizes on the button over the other font sizes

**Comment by Divide-By-0** [comment]
I don't like that we have to define new files for light and dark icons; seems like code duplication. I prefer my approach at https://github.com/Eeshau/zkemail/pull/5/files where we just pass the color mode as a prop. I've cherry picked and merge that commit -- I thought the way you had X and Github in particular working very differently (one with props and one with a bruteforced light logo) was not very consistent -- I've fixed it in my commit.

**Review by wryonik — APPROVED** [review]

**Comment by wryonik** [comment]
> I don't like that we have to define new files for light and dark icons; seems like code duplication.

Will make a separate issue for this

**Comment by Divide-By-0** [comment]
> > I don't like that we have to define new files for light and dark icons; seems like code duplication.
> 
> Will make a separate issue for this

I already fixed this in my pr
